patrones-chain-of-responsibility/README.md

```diff
@@ -0,0 +1,33 @@
+# Chain of responsability
+Es un patrón de diseño de comportamiento que te permite pasar solicitudes a lo largo de una cadena de manejadores. Al recibir una solicitud, cada manejador decide si la procesa o si la pasa al siguiente manejador de la cadena.
```

Comment:
https://medium.com/@alonsus91/patrones-de-dise%C3%B1o-chain-of-responsability-1460ee11c476
¿Está tomado de aquí? Deberían citar fuente.

Reply:
No profe ellos seguro lo sacaron de nuestro ReadME, nosotros nos basamos de la pagina Guru, aunque si deberiamos de haberla citado! Lo vamos a tener en cuenta en la proxima